dff/core/engine/conditions.py

```diff
@@ -21,13 +21,14 @@
 
 @validate_arguments
 def exact_match(match: Any, *args, **kwargs) -> Callable:
```

Comment:
Maybe we should specify `Callable` signature?
It should be something like `Callable[[Context, Actor, Any, Any], bool]` (AFAIK) here.